docs/zh_CN/FrameworkControllerMode.md

```diff
@@ -102,4 +102,8 @@ frameworkcontroller 模式中的 Trial 配置使用以下主键：
 
 ## 如何运行示例
 
-准备好配置文件后，通过运行 nnictl 来启动 Experiment。 在 Frameworkcontroller 上开始 Experiment 的方法与 Kubeflow 类似，可参考[指南](./KubeflowMode.md)了解更多信息。
\ No newline at end of file
+准备好配置文件后，通过运行 nnictl 来启动 Experiment。 在 Frameworkcontroller 上开始 Experiment 的方法与 Kubeflow 类似，可参考[指南](./KubeflowMode.md)了解更多信息。
+
+## 版本校验
+
+从 0.6 开始，NNI 支持查看版本，详情参考[这里](PAIMode.md)。
```

Comment:
how about use https://github.com/Microsoft/nni/blob/56ac616f4a777a15b64e35670293d21cc693ddf1/docs/zh_CN/PAIMode.md#%E7%89%88%E6%9C%AC%E6%A0%A1%E9%AA%8C instead.

Reply:
you can change en-us first, and it will be in zh-cn.